near-sdk-macros/src/lib.rs

```diff
@@ -643,3 +643,63 @@ pub fn derive_event_attributes(item: TokenStream) -> TokenStream {
         )
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
@shariffdev 

1. This place does not make sense for these tests. Let's move them to `near-sdk/tests/`
2. I would like to encourage you to consider looking into cargo-near implementation of abi extraction to avoid sh script running, etc: https://github.com/near/cargo-near/blob/48c7b48fa0f7f3b4afda447ba0b04cce3609b0a9/cargo-near/src/commands/abi_command/abi.rs#L31

Reply:
Updated